src/CamelKNodeProvider.ts

```diff
@@ -248,17 +257,34 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
 // simple tree node for our integration view
 export class TreeNode extends vscode.TreeItem {
 	type: string;
+	status: string;
 
 	constructor(
 		type: string,
 		label: string,
+		status: string,
 		collapsibleState: vscode.TreeItemCollapsibleState
 	) {
 		super(label, collapsibleState);
 		this.type = type;
-		this.iconPath = {
-			light: path.join(__filename, '..', '..', 'resources', 'round-k-transparent-16.svg'),
-			dark: path.join(__filename, '..', '..', 'resources', 'round-k-transparent-16.svg')
-		};
+		this.status = status;
+		this.iconPath = this.getIconForPodStatus(this.status);
+		this.tooltip = `Status: ${this.status}`;
+	}
+
+	getIconForPodStatus(status: string):  object {
+		let newIcon : object;
+		if (status.toLowerCase().startsWith("running")) {
+			newIcon = { 
+				light: vscode.Uri.file(path.join(__dirname, "../resources/round-k-transparent-16-running.svg")),
```

Comment:
light and dark theme are using the same image? Maybe we can get rid of the 2 cases then?